zombienet_tests/smoke/0003-deregister-register-validator-smoke.feature

```diff
@@ -0,0 +1,25 @@
+Description: Deregister / Register Validator Smoke
+Network: ./0003-deregister-register-validator-smoke.toml
+Creds: config
+
+alice: is up
+bob: is up
+charlie: is up
+dave: is up
+
+# ensure is in the validator set
+dave: reports polkadot_node_is_parachain_validator is 1 within 240 secs
```

Comment:
We to check both `polkadot_node_is_active_validator` and `polkadot_node_is_parachain_validator`